PostListHandler.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'; 
+import { posts } from './posts.js';
+import { users } from './users.js';
+import PostHandler from './PostHandler.js';
+ 
+function PostsListHandler() {
+  const postsList = [];
+  posts.map(post => {
+    const postTitle = post.title;
+    const postBody = post.body;
+    const postUser = users.find(user => user.id === post.userId);
+    const userName = postUser.name;
+    const userAddress = postUser.address.city;
+    const userEmail = postUser.email;
+    const postId = post.id;
```

Comment:
It is ok, but do you really need 7 variables? Also, for better readability, you can use destruction to create them